src/app/main_menu/option.rb

```diff
@@ -0,0 +1,19 @@
+class Options
+  def self.display
+    puts 'Welcome to the Library App'
+    puts 'Enter an option:'
+    puts '1 - List all books'
+    puts '2 - List all labels'
+    puts '3 - list of albums'
+    puts '4 - list_games'
+    puts '5 - list_authors'
+    puts '6 - genre list'
+    puts '7 - Add a book'
+    puts '8 - add music'
+    puts '9 - select genre'
+    puts '10 - add genre'
+    puts '11 - add game'
+    puts '12 - add author'
+    puts '0 - Exit'
+  end
```

Comment:
Good job so far, but I was left surprised by the result I got when I entered option 7 to add a book, but logically, the code written for that option is for adding music. In a sense:

- [ ] Kindly re-arrange the order of instruction to match the exact code logic written for that particular option as it's currently misleading. Kindly take a look at the screenshot below and see the option I entered and the instructions and results I got and how they are different.

![image](https://user-images.githubusercontent.com/50941074/205283319-5365aea2-a149-4690-8672-ab9bc1a30f50.png)

Option 8 should be option 7 and option 7 should be option 8. Please re-run the app and prove me wrong 👍